container-guide/adoc/containers-bci.adoc

```diff
@@ -150,6 +150,11 @@ dotnet-sdk::
   * URL: https://registry.suse.com/static/bci/dotnet-sdk
   * Comes with the .NET and ASP.NET SDK from Microsoft as well as the Microsoft .NET repository.
 
+[[sec-application-bci]]
+=== Application {bcia}s
+
+Application {bcia}s are {bcia}-Base container images that include specific applications, such as the PostgreSQL database and the Grafana analytics and monitoring solution. Application {bcia}s are available in the dedicated section of https://registry.suse.com[{suseregistry}].
```

Comment:
We do not ship Grafana on registry.suse.com, so that's a bad example imho. How about PCP instead?

```suggestion
Application {bcia}s are {bcia}-Base container images that include specific applications, such as the PostgreSQL database and the Grafana analytics and monitoring solution. Application {bcia}s are available in the dedicated section of https://registry.suse.com/#apps[{suseregistry}].
```

Reply:
Hm, I would personally disagree here, as I think it should be as simple for the user as possible. But you're right about the url not being stable. So your call.

I'll however insist on not using grafana as the example here.

Reply:
yes let's replace Grafana by: "Performance Co-Pilot (PCP) a system-level performance analysis toolkit for collecting and analyzing metrics in real-time or using historical data", and use the following URL: https://registry.suse.com/#apps